src/library/scala/collection/mutable/ListBuffer.scala

```diff
@@ -34,13 +33,15 @@ import scala.runtime.Statics.releaseFence
   *  @define mayNotTerminateInf
   *  @define willNotTerminateInf
   */
+@SerialVersionUID(3L)
```

Comment:
I think this should be explicitly set to the automatic UID generated before this PR.

Reply:
```
scala-launch 2.12.3 -nc -e 'println(java.io.ObjectStreamClass.lookup(classOf[scala.collection.mutable.ListBuffer[_]]).getSerialVersionUID)'
3419063961353022662
```

Reply:
I just realised this is the wrong UID, since this is for 2.**12**.3. I'll fix

Reply:
Oh sorry for the confusion!